### PR TITLE
:lock: Dependency update

### PR DESCRIPTION
**Pull request by w3rixie**
Patch Microsoft Identity Denial of service vulnerability CVE-2024-21319
Refference: GHSA-8g9c-28fc-mcx2

### CODE REVIEW

**Review by NickCraver — COMMENTED** [review]
Dangit, I added comments to this immediately but never submitted the review - my fault!

File: src/Opserver.Web/Opserver.Web.csproj [hunk 1]
```diff
@@ -9,7 +9,7 @@
     <PackageReference Include="BuildBundlerMinifier" Version="3.2.449" PrivateAssets="all" />
     <PackageReference Include="BuildWebCompiler" Condition="'$(OS)' == 'Windows_NT'" Version="1.12.405" PrivateAssets="all" />
     <PackageReference Include="MiniProfiler.AspNetCore.Mvc" Version="4.2.22" />
-    <PackageReference Include="Microsoft.IdentityModel.Protocols.OpenIdConnect" Version="6.15.0" />
+    <PackageReference Include="Microsoft.IdentityModel.Protocols.OpenIdConnect" Version="6.34.0" />
```
[the remark this answers is not in the document]
Comment (w3rixie) [reply src/Opserver.Web/Opserver.Web.csproj:12]: Latest, is greatest in my books. Will do.

**Comment by w3rixie** [comment]
> Dangit, I added comments to this immediately but never submitted the review - my fault!

All good. JIT for my build script to run.

**Comment by w3rixie** [comment]
Does this suit?
@NickCraver , did you submitted the review this time? 😄

**Comment by NickCraver** [comment]
Apologies, I only get weekend time for many things these days - thanks for the tweaks here!

**Pull request merged**